Week2/homework/exercise1.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+let myString = "hello,this,is,a,difficult,to,read,sentence";
+///the sentence will show up with comas between all the words
+console.log("the length of this sentence is " + myString.length);
+///will show the length of the sentence
+comaoff = myString.replace(/,/g," ");
+///now we gonna show the sentence without the comas
```

Comment:
You missed declaring the `comaoff` variable here before using it. Please take care of it as it is very important that you declare variables first before you use them. Also, I would suggest using a better name than `comaoff` as it doesn't fulfil two criterias:
1. Variable names should be semantic and describe the value they are holding. 
2. If a name consists of multiple words then camelCase should be used to declare that variable i.e comaOff